### ISSUE TITLE
Contrast missing

### ISSUE THREAD

**Issue by eglacias** [HallieBot/battibar#2]
Find a white screen it needs to be dark, on a black screen it needs to be light. Therefore it needs contrasting colors for each of these. Without these, it becomes unreadable on either one of the two color contrasts

Needs  to follow the iOS status bar which switches back-and-forth according to the background contrast
